### PR TITLE
fixed KREW install command

### PR DESCRIPTION
**Pull request by FiloGiova**
# Description

I have changed only the KREW install command that has been modified on official documentation after the birth of M1 processors.

### CODE REVIEW

File: infrastructure/identity-provider/README.md [hunk 1]
```diff
@@ -118,15 +118,16 @@ Here there is the commands for Linux (Bash/Zsh):
...
+ OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+ ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+ KREW="krew-${OS}_${ARCH}" &&
+ curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/${KREW}.tar.gz" &&
+ tar zxvf "${KREW}.tar.gz" &&
+ ./"${KREW}" install krew
 )
  ```
-
+Replace amd64 with arm64 in the ARCH field if you have a Macbook with M1 processsor.
```
Comment (giorio94) [inline infrastructure/identity-provider/README.md:130]: Isn't this automatically done by the command above?

Comment (FiloGiova) [reply infrastructure/identity-provider/README.md:130]: Yeah, sorry

**Review by giorio94 — APPROVED** [review]
/lgtm Just squash the two commits before merging

**Pull request merged**